package.json

```diff
@@ -10,9 +10,7 @@
     "react-dom": "^16.13.1",
     "react-router": "^5.2.0",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "3.4.3",
-    "@material-ui/core": "^4.12.3",
-    "@material-ui/icons": "^4.11.2"
+    "react-scripts": "3.4.3"
```

Comment:
this is the correct way to fix the related jsdom issues on the tests, good catch